Gordon360/Controllers/RecIM/MatchesController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Services.RecIM;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Gordon360.Controllers.RecIM
+{
+    [Route("api/recim/[controller]")]
+    [AllowAnonymous]
+    public class MatchesController : GordonControllerBase
+    {
+        private readonly IMatchService _matchService;
+
+        public MatchesController(IMatchService matchService)
+        {
+            _matchService = matchService;
+        }
+
+        /// <summary>
+        /// Not implemented (yet) might be removed
+        /// </summary>
+        /// <param name="day"></param>
+        /// <param name="active"></param>
+        /// <returns></returns>
+        /// <exception cref="NotImplementedException"></exception>
+        [HttpGet]
+        [Route("")]
+        public ActionResult<IEnumerable<MatchViewModel>> GetMatches([FromQuery] DateTime? day, bool active)
```

Comment:
```suggestion
        public ActionResult<IEnumerable<MatchViewModel>> GetMatches([FromQuery] DateTime? day, [FromQuery] bool active)
```
I believe you would have to specify for each parameter.

Reply:
Actually, now that I am thinking about it, you don't need to specify for either of them, it is the default that the parameter comes from the query string. Whatever you choose to do, please do the same for all get parameters.